Sources/BraintreeUIKit/Helpers/UIFont+BTUIK.m

```diff
@@ -0,0 +1,89 @@
+#import "UIFont+BTUIK.h"
+
+@implementation UIFont (BTUIK)
+
++ (UIFont *)bodyFontForFontFamily:(NSString *)fontFamily useStaticSize:(BOOL)useStaticSize {
+    UIFont *customFont;
+    if (fontFamily) {
+        customFont = [UIFont fontWithName:fontFamily size:UIFont.labelFontSize];
+    }
+
+    if (customFont) {
+        if (useStaticSize) {
+            return customFont;
+        } else {
+            return [[UIFontMetrics metricsForTextStyle:UIFontTextStyleBody] scaledFontForFont:customFont];
+        }
+    } else {
+        if (useStaticSize) {
+            return [UIFont systemFontOfSize:UIFont.labelFontSize];
+        } else {
+            return [UIFont preferredFontForTextStyle:UIFontTextStyleBody];
+        }
+    }
```

Comment:
There are four cases for each font:
- Custom font with dynamic type disabled
- Custom font with dynamic type enabled
- System font with dynamic type disabled
- System font with dynamic type enabled